documentation/modules/proc-send-receive.adoc

```diff
@@ -29,7 +29,8 @@ Use the output as the host name in the following step.
 
 . Run the client:
 +
-[options="nowrap",subs=attributes+]
+[options="nowrap",subs="+quotes,attributes"]
 ----
-python client-example1.py -u amqps://user1:password@messaging.example1.com:443/myqueue
+python client-example1.py -u amqps://user1:password@_messaging.example1.com_:443/myqueue
```

Comment:
I think this might break the console help page which replaces the hostname with the actual hostname.

Reply:
@jenmalloy i will build and check the console, so hold off merging until i've verified.

Reply:
@jenmalloy Help page looks good. Ship it!

Reply:
@lulf Thank you!